docs/manual/faq/_index.en.md

```diff
@@ -161,3 +161,14 @@ delete the file `task.json`.
 
 After that, the Contao Manager should run again.
 {{% /expand %}}
+
+{{% expand "Can I rename the .phar file?" %}}
```

Comment:
{{% expand "Can I rename the `.phar` file?" %}}

Reply:
Done with »...«